weed/filer/elastic/v7/elastic_store.go

```diff
@@ -66,15 +72,66 @@ func (store *ElasticStore) CommitTransaction(ctx context.Context) error {
 func (store *ElasticStore) RollbackTransaction(ctx context.Context) error {
 	return nil
 }
+
 func (store *ElasticStore) KvDelete(ctx context.Context, key []byte) (err error) {
-	return filer.ErrKvNotImplemented
+	id := fmt.Sprintf("%x", md5.Sum(key))
+	deleteResult, err := store.client.Delete().
+		Index(indexKV).
+		Type(indexType).
+		Id(id).
+		Do(context.Background())
```

Comment:
there is already a `ctx` parameter. No need to use `context.Background()`. Same for other places.

Reply:
it can be without indexing if totally no need to query.

Reply:
Only need to fetch the value by the key.